MotionCommand desired_motion
---
int32 error_code
int32 SUCCESSFUL = 0
int32 ERROR = -1
int32 INVALID_TRAJECTORIES = -2
